firefox_desktop/views/history_sidebar_clients_daily.view.lkml

```diff
@@ -0,0 +1,86 @@
+view: history_sidebar_clients_daily {
+
+  sql_table_name: `moz-fx-data-shared-prod`.telemetry_derived.clients_daily_joined_v1 ;;
+
+  dimension: client_id {
+    type: string
+    sql: ${TABLE}.client_id ;;
+    hidden: yes
+  }
+
+  dimension: submission_date {
```

Comment:
you can optionally extend this field to allow for timeframes other than dates, if you want.

[here is an example](https://github.com/mozilla/looker-hub/blob/main/firefox_desktop/views/clients_daily_joined_table.view.lkml#L1968-L1981)f

Reply:
Thanks! Added